documentation/docs/welcome.md

```diff
@@ -0,0 +1,55 @@
+---
+description: 'Stronghold is an open-source software library can be used to protect any digital secret. It is a secure
+database for working with cryptography, which ensures that secrets are never revealed - but can be used according to
+best practices.'
+image: /img/Banner/banner_stronghold.png
+keywords:
+
+- welcome
+- open-source
+- secure
+- secrets
+- database
+- p2p
+
+---
+
+# Welcome
+
+![Stronghold](/img/Banner/banner_stronghold.png)
+
+Stronghold is an open-source software library that was originally built to protect IOTA Seeds, but can be used to
+protect any digital secret.
+
+Stronghold is a secure database for working with cryptography, which ensures that secrets (like private keys) are never
```

Comment:
We could replace the second and third strongholds for "It provides" or "It is", but since it's the welcome page I thought it'd be best to repeat the product name a bit.